packages/ui-kit/src/design-system.js

```diff
@@ -51,6 +51,7 @@ export const colors = {
     borderForReleaseNoteTypeFix: '#b866ff',
     // Links
     link: customProperties.colorPrimary25,
+    linkChildSectionNavigationBackground: '#f2f2f2',
```

Comment:
Please use the correct design token / property

Reply:
Right, thanks 👍